sirepo/package_data/static/json/flash-schema.json

```diff
@@ -1274,6 +1277,9 @@
             "fl_fsConstFlameSpeed": ["Constant Flame Speed", "Float", 1.0, "fl_fsConstFlameSpeed"],
             "fl_fsConstFlameWidth": ["Constant Flame Width", "Float", 1.0, "fl_fsConstFlameWidth"]
         },
+        "physics:sourceTerms:Heatexchange": {
+            "useHeatExchange": ["Use Heat Exchange", "Boolean", "0", "Use heat exchange"]
```

Comment:
I see useHeatExchange was added to the schema, but it isn't in a view for editing. That could be intentional, but I'm just making sure.

Reply:
That was unintentional. https://github.com/radiasoft/sirepo/pull/2603/commits/9ed9f1c9c3073e4f56453174f1364c4de88884b9